tests/DCPS/XTypes/XTypes.idl

```diff
@@ -0,0 +1,141 @@
+#include <tao/OctetSeq.pidl>
+
+@topic
+struct Property_1 {
+  @key long key;
+  long value;
+};
+
+// TODO: remove if not needed
+@topic
+struct Property_2 {
+  @key long key;
+  string value;
+};
+
+// Final structues
+
+@topic
+@final
+@OpenDDS::data_representation(XCDR2)
```

Comment:
The `data_representation` annotations can be removed

Reply:
In all definitions?

Reply:
Yes